camelot/admin/action/list_filter.py

```diff
@@ -375,15 +376,15 @@ def get_type_clause(self, text, field_attributes):
             return sql.and_(self.attribute>=float_value-delta, self.attribute<=float_value+delta)
         except utils.ParsingError:
             pass
-    
+
     def value_to_string(self, value, admin):
         field_attributes = admin.get_field_attributes(self.attribute.key)
+        field_attributes = {h:copy.copy(v) for h,v in field_attributes.items()}

```

Comment:
ik denk niet dat je hier een copie vd value moet nemen ?

Reply:
Hmm idd, ik ging er uit gewoonte vanuit dat er een deep copy nodig is, maar het is hier idd een sub-dict voor een enkel attribute, dus een shallow copy is genoeg. Fixed in [951249c](https://github.com/v-finance/camelot/pull/409/commits/951249c8515b6a9348f305c2333035696037ad4f).